plugins/CoreVisualizations/Visualizations/HtmlTable/AllColumns.php

```diff
@@ -64,4 +64,14 @@ public function beforeGenericFiltersAreAppliedToLoadedDataTable()
             $properties->columns_to_display = $columnsToDisplay;
         });
     }
+
+    public function beforeLoadDataTable()
+    {
+        $this->requestConfig->request_parameters_to_modify['pivotBy'] = null; // always unset pivotBy
```

Comment:
this fixes another issue that was occurring when switching all columns view after pivot was activated.